api/app/signals/apps/signals/management/commands/load_categories.py

```diff
@@ -0,0 +1,62 @@
+import json
```

Comment:
Move this file to the `signals.apps.dataset` app, to match with the location of the tests.